kafka-ui-react-app/tsconfig.json

```diff
@@ -17,8 +17,9 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react",
-    "baseUrl": "src"
+    "jsx": "react-jsx",
```

Comment:
Currently the build is failing because expected values for this option are `preserve`, `react-native`, `react`